python/Makefile

```diff
@@ -1,3 +1,11 @@
+SHELL := /bin/bash
+
+ifndef SCALA_CLASS_PATH
+$(error SCALA_CLASS_PATH for python tests is not set. Please check out \
+the top-level Makefile on how to source scala_classpath_for_python.sh)
```

Comment:
Nice 👍 

For the hasty (like me), `spark_session.py` could also have an assertion with a helpful message like this?